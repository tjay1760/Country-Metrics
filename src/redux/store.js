import { configureStore } from '@reduxjs/toolkit';
import countryReducer from './countries/countrySlice';

const store = configureStore({
  reducer: {
    countries: countryReducer,
  },
});

export default store;
