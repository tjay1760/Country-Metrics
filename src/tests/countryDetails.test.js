import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../redux/store';
import CountryDetails from '../components/countryDetails';

describe('CountryDetails component', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(
      <Provider store={store}>
        <MemoryRouter initialEntries={['/country/countryName']}>
          <CountryDetails />
        </MemoryRouter>
      </Provider>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
