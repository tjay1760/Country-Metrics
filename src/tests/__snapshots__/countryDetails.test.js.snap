// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CountryDetails component should match snapshot 1`] = `
<DocumentFragment>
  <div>
    ...
  </div>
</DocumentFragment>
`;
